frontend/src/features/properties/list/PropertyListView.scss

```diff
@@ -0,0 +1,56 @@
+$header-height: 72px;
+$footer-height: 46px;
+$navbar-height: 45px;
+$filter-height: 76px;
+$contentheading-height: 60px;
```

Comment:
I had to copy these values from the map view page. I've seen them in the property forms.

I don't like copy-paste code but didn't have time to think of a better way. After PoC will take the time to think about this